### PR TITLE
Disallow empty transfers

### PR DESCRIPTION
**Pull request by Lipt0nas**

### CODE REVIEW

File: test/scenarios.py [hunk 1]
```diff
@@ -3353,6 +3353,28 @@
...
+    Scenario(
+        "scenario17-1",
+        "Try to send an empty directory, expect error",
+        {
+            "DROP_PEER_REN": ActionList(
+                [
+                    action.Start("DROP_PEER_REN"),
+                    action.ConfigureNetwork(),
+                    action.WaitForAnotherPeer(),
+                    action.NewTransferFails("DROP_PEER_STIMPY", "/tmp/empty-dir"),
```
Comment (LukasPukenis) [inline test/scenarios.py:3365]: Can we add a nested deep directory as well:
```
/tmp/empty-dir
/tmp/empty-dir/dir/
/tmp/empty-dir/dir/dir
```

and upon sending `/tmp/empty-dir` expect an error?

**Review by LukasPukenis — DISMISSED** [review]
nice!

File: test/scenarios.py (hunk 1 above)
Comment (LukasPukenis) [reply test/scenarios.py:3365]: Just an additional check

Comment (Lipt0nas) [reply test/scenarios.py:3365]: Done

**Review by LukasPukenis — APPROVED** [review]

**Pull request merged**